inspire_dojson/hep/rules/bdFFT.py

```diff
@@ -46,8 +46,7 @@ def _is_hidden(value):
         return 'HIDDEN' in [val.upper() for val in force_list(value.get('o'))] or _get_source(value) == 'arxiv' or None
 
     def _is_figure(value):
-        figures_extensions = ['.png']
-        return value.get('f') in figures_extensions
+        return value.get('f', "").endswith(".png")
```

Comment:
Can we also check for "source": "plot" as I noticed on the ones I've checked all of them have "plot" in source

Reply:
Ok I just found record with fake documents which do not have plot in source and file extension is .gif and .eps